### PR TITLE
add notFoundPage

### PR DESCRIPTION
**Pull request by OleksiiKnihin**
added notFoundPage and changed img for this page

### CODE REVIEW

File: src/pages/notFoundPage/NotFoundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './NotFoundPage.scss';
```
Comment (Kasprukov) [inline src/pages/notFoundPage/NotFoundPage.tsx]: We agreed to use styles like this:
```suggestion
import classes from './NotFoundPage.module.scss';
```

File: src/pages/notFoundPage/NotFoundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// import notFoundImg from '../../../public/img/page-not-found.png';
+
+export const NotFoundPage: React.FC = () => {
+  return (
+    <div className="pageNotFound__wrapper">
```
Comment (Kasprukov) [inline src/pages/notFoundPage/NotFoundPage.tsx]: Do this changes for all classes.
```suggestion
    <div className={classes.pageNotFound__wrapper}>
```

**Review by Kasprukov — CHANGES_REQUESTED** [review]
Good job! 🔥

**Review by FaiHamid — APPROVED** [review]
Well done))

**Review by Kasprukov — APPROVED** [review]
Well done! 🔥

**Pull request merged**